src/types/posts.ts

```diff
@@ -111,7 +112,7 @@ export type MessageHistory = {
 
 export type PostsState = {
     posts: IDMappedObjects<Post>;
-    postsReplies: {[x in $ID<Post>]: number};
+    postsReplies: { [x in $ID<Post>]: number };
```

Comment:
```suggestion
    postsReplies: {[x in $ID<Post>]: number};
```
nit: I don't think eslint/tsc checks for this in particular, but the normal convention (mattermost-webapp/redux) is no spaces around inline object notation 1/5